x-pack/plugins/rollup/server/routes/api/indices.js

```diff
@@ -21,16 +21,13 @@ export function registerIndicesRoute(server) {
         const capabilities = await callWithRequest('rollup.capabilities', {
           indices: '_all'
         });
-        reply(mapCapabilities(capabilities, index));
+        reply(getIndexNameToCapabilitiesMap(capabilities, index));
       } catch(err) {
         if (isEsError(err)) {
           return reply(wrapEsError(err));
         }
         reply(wrapUnknownError(err));
       }
     },
-    // config: {
```

Comment:
Would like to still keep this here as a reminder - I think we still need some pre-routing checks :)